### ISSUE TITLE
What exactly does "headless" mean in your case.

### ISSUE THREAD

**Issue by MontyTHall** [jonnenauha/ogre-headless-renderer#1]
I'd like to render a scene to a texture.  Can you do this w/ your headless rendering system?  If not, it's not exactly what I expected then.  What's it intended use if you can't load textures or render to target?

Thanks

**Comment by jonnenauha** [comment]
Its right there in the readme https://github.com/jonnenauha/ogre-headless-renderer#use-case

The idea is to use this for example in headless non-rendering window setup on a server for example. You may use Ogre scene but cannot do what you are suggesting. There is no opengl/directx initialized that could do that, there for no opengl or directx dependency either.

**Comment by jonnenauha** [comment]
You could just use the Ogre provided renderers to do this? What are the restrictions for not using them in your case?

**Comment by MontyTHall** [comment]
Yeah, I saw that.  I'm trying to think under what circumstance would having
only the scene graph would be useful.  It's almost like it's not "ogre
headless" but "ogre rendering stub".  I've not written a game server, but
it's not clear why a scene graph would be needed.

Sorry to bother you, just thinking out loud.  But yes, I can't use this and
am looking for other avenues.  I believe Xvfb may help.  I'm also looking
into modifying the existing glx renderer to go headless.

Thanks
